day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,13 @@
+# Caesar Cipher
+
+class CaesarCipher
+
+  def encrypt(string, key)
+    string_to_ascii_array = string.chars.map {|char| char.ord}
+    shifted = string_to_ascii_array.map {|char| char + key}
+    shifted.map {|char| char.chr }.join(', ')
+  end
```

Comment:
WARNING: Never copy and paste code as your own. If work is still in progress and copied code is for reference, make a comment to that effect.

Reply:
So I have a question about this if/when you have time. I know there are usually multiple ways to solve problems but programmers typically settle on one. In situations like these are you saying it's ok to use code already written by others so long as I leave a comment saying so? Sorry I feel like this is a tricky area as I'm sure programmers use code that's already written regularly. If not, I'm sure they use code written by others as inspiration for their own. Is what you stated above your best advice for how to handle situations like these going forward?